APMrover2/mode.cpp

```diff
@@ -394,7 +394,7 @@ void Mode::navigate_to_waypoint()
     desired_speed = calc_speed_nudge(desired_speed, g2.wp_nav.get_reversed());
     calc_throttle(desired_speed, true);
 
-    float desired_heading_cd = g2.wp_nav.wp_bearing_cd();
+    float desired_heading_cd = g2.wp_nav.oa_wp_bearing_cd();
```

Comment:
Is this always available? What happend is one disables obstacle avidance compilation ?

Reply:
https://github.com/ArduPilot/ardupilot/blob/c445362fae56e0e1d7fd7e3448a8a7e5bf9da41f/libraries/AR_WPNav/AR_WPNav.cpp#L327

It is correct if it is disabled